VERSION

```diff
@@ -1,4 +1,6 @@
-v9.1.2
+v9.1.3
+
+v9.1.3 Fix for generated code supporting filtering on an enum
```

Comment:
It's generally best practice when adding a new line here to remove the bottom one so our history here doesn't get too large.

Reply:
But it looks like we haven't been doing that here?

Reply:
resolved